core/trino-spi/src/test/java/io/trino/spi/predicate/BenchmarkSortedRangeSet.java

```diff
@@ -169,6 +169,36 @@ private long benchmarkContainsValue(List<SortedRangeSet> dataRanges)
         return result;
     }
 
+    @Benchmark
+    public List<Integer> getOrderedRangesSmall(Data data)
+    {
+        return benchmarkGetOrderedRanges(data.smallRanges);
```

Comment:
could we parametrize number of ranges instead?

Reply:
this seems to be consistent with others here

Reply:
I know. You've added them ;)

Reply:
i know :)

Reply:
Would you like to refactor it?

Reply:
No, i'd prefer to keep it as it is, and consider overall improvement a good follow up candidate.

Reply:
>  and consider overall improvement a good follow up candidate.

makes sense. Most (all) other benchmarks parametrize dataset size if needed to reduce code duplication